modules/tiles/src/tileset/tile-3d.js

```diff
@@ -461,7 +478,7 @@ export default class TileHeader {
     this._centerZDepth = 0;
     this._screenSpaceError = 0;
     this._visibilityPlaneMask = CullingVolume.MASK_INDETERMINATE;
-    this._visible = false;
+    this._visible = null;
```

Comment:
What is the type of _visible? Can it be set to `null`? We have a typescript PR about to land.

Reply:
Would it work as optional `_visible?: boolean`? 
I used this to indicate when the visibility is not at all updated (headless, tile-covnerter) so the scheduler will still process the request.

Reply:
Well OK, I certainly wish we didn't need so many optional fields. The firmer we can type a field the better. More variations => more bugs.

Reply:
Well, another way would be to add a second boolean such as `headless` which would be falsified when visibility is updated, but I am not sure if it is more readable.
How about using `undefined` and defining in Typescript:
`_visible: boolean | undefined` ? I'm not sure what's the standard way to type variables that have an optional value (like this [Rust equivalent)](https://doc.rust-lang.org/std/option/) or [this one](https://gcanti.github.io/fp-ts/modules/Option.ts.html)).

Reply:
Sure. If it is only this field that needs three states, then `_visible?: boolean` would work as opposed to having two fields just for this. However if we need to return this to the `undefined` state at some point, I'd rather use `null` (`_visible: boolean | null`. 

If there were multiple fields that could be controlled by one common field that indicates whether things are initialized, then I'd personally separate it out.

This is all moot before we actually land the typescript branch.

Reply:
OK. I think then we can stick with `_visible?: boolean` for this one. Seems readable enough to me.